pkg/services/gotify/gotify.go

```diff
@@ -0,0 +1,95 @@
+package gotify
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/containrrr/shoutrrr/pkg/services/standard"
+	"github.com/containrrr/shoutrrr/pkg/types"
+)
+
+// Service providing teams as a notification service
+type Service struct {
+	standard.Standard
+	config *Config
+}
+
+// Initialize loads ServiceConfig from configURL and sets logger for this Service
+func (service *Service) Initialize(configURL *url.URL, logger *log.Logger) error {
+	service.Logger.SetLogger(logger)
+	service.config = &Config{}
+	err := service.config.SetURL(configURL)
+	return err
+}
+
+func buildURL(config *Config) (string, error) {
```

Comment:
Are there any other format requirements of the token? For example that it should be this and that length or only contain alphanumericals or whatever. If so, that would be nice to validate as well.

Reply:
Added some checks, based on what I've found in the Gotify code